addon/components/eosf-project-nav/component.js

```diff
@@ -0,0 +1,93 @@
+import Ember from 'ember';
+import DS from 'ember-data';
+
+import layout from './template';
+
+import permissions from '../../const/permissions';
+
+/**
+ * @module ember-osf
+ * @submodule components
+ */
+
+/**
+ * Project header navigation bar, with links to various sub-pages within a project
+ * Sample usage:
+ * ```handlebars
+ * {{eosf-project-nav
+ *   node=node
+ *   user=user}}
+ * ```
+ * @class eosf-project-nav
+ */
+export default Ember.Component.extend({
+    layout,
+    node: null,
+    user: null,
+
+    projectLabel: Ember.computed('node.category', function() {
+        let category = this.get('node.category');
+        return category === 'project' ? 'Project' : 'Component';
+    }),
+
+    isProjectContributor: Ember.computed('user', 'node', function() {
+        // Is the user a contributor on this node? (returns false if the user is not logged in)
+        // FIXME: This depends on OSF-6702, a known bug.
+        let node = this.get('node');
+        let userID = this.get('user.id');
+
+        return DS.PromiseObject.create({
+            promise: node.isContributor(userID)
+        });
+    }),
+
+    /**
+     * If this is a withdrawn registration, hide a block of buttons.
+     * @property minimalRegistrationView
+     */
+    minimalRegistrationView: Ember.computed.and('node.isRegistration', 'node.withdrawn'),
```

Comment:
Kind of nit-picky, but this computed's name seems to refer to the view state. Maybe `isWithdrawnRegistration` instead?